easybuild/easyconfigs/s/Score-P/Score-P-2.0.1_fix_score_configure.patch

```diff
@@ -0,0 +1,52 @@
+--- build-score/configure.orig	2016-05-11 09:21:03.716441529 +0200
```

Comment:
@geimer: please include a comment above this line to explain what this patch does, and mention yourself as an author

```
prevent build-score/configure from picking up $CFLAGS etc., backported from Score-P 2.0.2
author: Markus Geimer (JSC)
```